cdap-hbase-compat-1.2-cdh5.7.0/pom.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2015 Cask Data, Inc.
```

Comment:
Year.
Also should we update the year from-to of all the above files which you modified